libs/spice/Makefile

```diff
@@ -56,16 +58,18 @@ MESON_ARGS += \
 	-Dspice-common:generate-code=none \
 	-Dspice-common:tests=false
 
-define Package/libspice-server/install
+define Build/InstallDev
+	$(INSTALL_DIR) $(1)/usr/include/spice-server
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/spice-server/*.h $(1)/usr/include/spice-server
 	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libspice-server.so* $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libspice-server.* $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/spice-server.pc $(1)/usr/lib/pkgconfig
 endef
 
-define Build/InstallDev
+define Package/libspice-server/install
 	$(INSTALL_DIR) $(1)/usr/lib
-	$(INSTALL_DIR) $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/include/* $(1)/usr/include
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/* $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libspice-server.so* $(1)/usr/lib
```

Comment:
To be honest, I am not a fan of commit like this.  A bit messy and takes more time to check if there are unexpected changes.  Try not stuffing in another subject in the message body please.

Reply:
Split off to a separate commit.